sdk/cosmos/azure_data_cosmos/Cargo.toml

```diff
@@ -15,16 +15,16 @@ categories = ["api-bindings"]
 
 [dependencies]
 async-trait.workspace = true
-azure_core.workspace = true
+azure_core = "0.22.0"
```

Comment:
This is the wrong way, but perhaps necessary for right now. /cc @hallipr. The workspace dependency should be a version, while local dependencies should be path until after `azure_core` or whatever dependency ships (`dev-dependencies` should always be path going forward, but we'll solve that later).

Managing versions across many different crates will be difficult as we onboard hundreds. I don't want to have to fix all those when we get CG alerts. Most other languages have centralized dependencies.

Reply:
I'm fine promoting this to the workspace if that's what you'd prefer, now that we've released.

Reply:
Might be a good test, though until we GA nearly everything will end up being path.

Hmm, that said, maybe this is fine for now since we'll just need to change it back anyway because of that. @hallipr what do you think?

Reply:
Ok, why don't I leave this as specific to `azure_data_cosmos` for now and we can sort out our plan for workspace-wide dependencies separately. I'd really like to get this `.1` working and released because we're seeing a few customers notice the issues (or at least unintended side-effects, like the failed docs build).